BRAVE/docker-compose.prod.yml

```diff
@@ -0,0 +1,59 @@
+version: "3.8"
```

Comment:
New version of docker do not use `version` it has been deprecated and can be removed.